grafana-plugin/e2e-tests/globalSetup.ts

```diff
@@ -18,15 +18,9 @@ import {
   GRAFANA_VIEWER_USERNAME,
   IS_CLOUD,
   IS_OPEN_SOURCE,
+  OrgRole,
```

Comment:
This was removed in the past because the jest tests had an import issue if this was being used, but it was not complaining here directly. Not against removing it, but the issue might come again.